packages/cli/src/commands/start.ts

```diff
@@ -141,6 +141,7 @@ export class Start extends BaseCommand {
 					createReadStream(filePath, 'utf-8'),
 					replaceStream('/{{BASE_PATH}}/', n8nPath, { ignoreCase: false }),
 					replaceStream('/%7B%7BBASE_PATH%7D%7D/', n8nPath, { ignoreCase: false }),
+					replaceStream('/%257B%257BBASE_PATH%257D%257D/', n8nPath, { ignoreCase: false }),
```

Comment:
Could we add a test to prevent this bug from happening happening ?

Reply:
there are no tests for this file. Considering the amount of work required to refactor this file to be able to test it, I'd much rather spend that time fixing the vite config, and remove the two additional `replaceStream` calls here.

Reply:
Would an integration test making sure the font files load make sense?

Reply:
by integration I mean e2e*

Reply:
since this issue only happens for favicon and fonts, we could update one of the e2e tests to assert that the favicon url was loaded. besides that I can't think of any other reliable way to test this via e2e.

Reply:
Would you please try that then if it makes sense?

Reply:
Tried to add testing the favicon, but seems not to load with or without the fix, so just let it go for now.